fractal-puppy/Engine
# Feature/dropping items

**Objectives**:
- When an item dragged from the inventory to the world, the character doesn't move
- When an item is dropped, it reappears in the world (as a temporary measure (?), it reappears in the same position as the player)
- The item description panel disappears when an item is dropped

**Test**:
- Pick up an item
- Move
- Open the inventory menu
- Drag the item from the inventory to the world (the character should not move)
- Drop the item (the item should appear at the character's feet and the description panel should disappear)

**Notes**:
- In uneven terrain (stairs,...) the item bag may cause a graphic bug as it appears overlapping that terrain.
- The GIF below appears laggy because the GPU was working under heavy load due to unrelated tasks.

![Drop](https://user-images.githubusercontent.com/26639060/65207871-8c46ab00-da93-11e9-9472-4f87ec6209c4.gif)
